docs: misc changes triggered while revisiting translations (smoe:docs29_mac_104)

smoe opened the pull request:
Not too much left.

hansu commented on docs/src/code/writing-tests.adoc:
```diff
@@ -34,8 +33,8 @@ _test.hal_ below the directories specified on the command line, or under
 
-The _runtests_ script accepts the following arguments, see the output
-from `scripts/runtests -h` for the authoritative list:
+.Arguments acceppted by The _runtests_ script, see the output
```
```suggestion
.Arguments accepted by The _runtests_ script, see the output
```

hansu commented on docs/src/ladder/classic-ladder.adoc:
```diff
@@ -395,46 +395,46 @@ Represent up/down counters.
+* `%C`__xx__`.E` - Counter _xx_ empty overflow (Boolean, read only)
+* `%C`__xx__`.F` - Counter _xx_ full overflow (Boolean, read only)
+* `%C`__xx__`.V` - Counter _xx_ current value (integer, read or write)
+* `%C`__xx__`.P` - Counter _xx_ preset (integer, read or write)
```
```suggestion
* '%C'__xx__`.P` - Counter _xx_ preset (integer, read or write)
```
Please leave them in italic to keep it consistent

hansu commented on docs/src/ladder/classic-ladder.adoc:
```diff
@@ -395,46 +395,46 @@ Represent up/down counters.
+* `D` - down count (input) will subtract one from the count.
+* `E` - under flow (output) will be true when the count rolls over from 0 to 9999.
+* `D` - done (output) will be true when the count equals the preset.
+* `F` - overflow (output) will be true when the count rolls over from 9999 to 0.
```
```suggestion
* 'F' - overflow (output) will be true when the count rolls over from 9999 to 0.
```
Please leave them in italic to keep it consistent with the rest

hansu reviewed (changes requested):
And please rebase to resolve the conflict.

smoe commented:
@hansu, I messed up, performed the rebase without first pulling your changes. Please kindly have a look at [a1359dd](https://github.com/LinuxCNC/linuxcnc/pull/2592/commits/a1359dddab1fc8cc15fc7f17336d7213a13da452) to ensure that this addresses all in the way you wanted this to be addressed.

hansu commented:
> @hansu, I messed up, performed the rebase without first pulling your changes. Please kindly have a look at [a1359dd](https://github.com/LinuxCNC/linuxcnc/pull/2592/commits/a1359dddab1fc8cc15fc7f17336d7213a13da452) to ensure that this addresses all in the way you wanted this to be addressed.

I haven't done any changes to your branch. But the change from my first comment seems got lost.

smoe commented:
> > @hansu, I messed up, performed the rebase without first pulling your changes. Please kindly have a look at [a1359dd](https://github.com/LinuxCNC/linuxcnc/pull/2592/commits/a1359dddab1fc8cc15fc7f17336d7213a13da452) to ensure that this addresses all in the way you wanted this to be addressed.
> 
> I haven't done any changes to your branch. But the change from my first comment seems got lost.

Those changes from your comments I have meant to have reintroduced in [a1359dd](https://github.com/LinuxCNC/linuxcnc/pull/2592/commits/a1359dddab1fc8cc15fc7f17336d7213a13da452) . I had felt just fine wrt having addressed all issues, just I was not sure if I was interpreting all your changes correctly. What change from your first comment are you referring to?

hansu commented on docs/src/code/writing-tests.adoc:
```diff
@@ -34,8 +33,7 @@ _test.hal_ below the directories specified on the command line, or under
 
-The _runtests_ script accepts the following arguments, see the output
-from `scripts/runtests -h` for the authoritative list:
+.Arguments acceppted by The _runtests_ script, see the output from `scripts/runtests -h` for the authoritative list.
```
This one:

```suggestion
.Arguments accepted by the _runtests_ script, see the output from `scripts/runtests -h` for the authoritative list.
```

smoe commented:
Accepted your suggested change, and also pulled it into my local repository, just to be sure :)

smoe commented:
There is no docs29_mac_105 branch on my side, so this may indeed be the last change from my side that would not contribute some content. Happy!

hansu commented:
> There is no docs29_mac_105 branch on my side, so this may indeed be the last change from my side that would not contribute some content. Happy!

We will see ;-)
